src/side-nav/side-nav.jsx

```diff
@@ -0,0 +1,370 @@
+import React from 'react';
+import { AutoPrefix, Transitions } from '../styles';
+import Overlay from '../overlay';
+import Menu from '../menus/menu';
```

Comment:
Why did you use menus/menu ? Is it just to be able to pass children to it? I think menus/menu is a complex component that has it's own behavior. That's an overhead to use is just to embed some menu items. This is probably why LeftNav used a much lighter menu/menu (NestedMenuItem) component.